Codemine/Extensions/String+Range.swift

```diff
@@ -17,7 +17,7 @@ public extension String {
      
      - returns: true if the string contains that range, false otherwise
      */
-    private func containsRange(_ range: Range<Index>) -> Bool {
+    private func contains(_ range: Range<Index>) -> Bool {
```

Comment:
I think we should make the parameter name non optional, otherwise it's a bit unclear.

EDIT: Probably not, the guidelines are fine with this :P